Resources/views/CRUD/submission_delete.html.twig

```diff
@@ -0,0 +1,52 @@
+{#
+
+This file is part of the Sonata package.
+
+(c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+
+For the full copyright and license information, please view the LICENSE
+file that was distributed with this source code.
+
+#}
+
+{% extends base_template %}
+
+{%- block actions -%}
+    {% include '@SonataAdmin/CRUD/action_buttons.html.twig' %}
+{%- endblock -%}
+
+{%- block tab_menu -%}
+    {{ knp_menu_render(admin.sidemenu(action), {
+        'currentClass': 'active',
+        'template': get_global_template('tab_menu_template')
+    }, 'twig') }}
+{%- endblock -%}
+
+{% block content %}
+    <div class="sonata-ba-delete">
+
+        <div class="box box-danger">
+            <div class="box-header">
+                <h3 class="box-title">{{ 'title_delete'|trans({}, 'SonataAdminBundle') }}</h3>
+            </div>
+            <div class="box-body">
+                {{ 'message_delete_confirmation'|trans({'%object%': admin.toString(submission)}, 'SonataAdminBundle') }}
+            </div>
+            <div class="box-footer clearfix">
+              <form method="POST" action="{{ path('admin_pirastru_formbuilder_formbuilder_submission_delete', { "id": form.id, "submission_id": submission.id }) }}">
```

Comment:
You should never generate controller paths directly if they're part of an Admin class. See [Routing usage documentation](https://docs.sonata-project.org/projects/SonataAdminBundle/en/4.x/reference/routing/#routing-usage).